client/actions/authenticationAction.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import jwtDecode from 'jwt-decode';
+import setAuthorizationToken from '../util/util';
+import { SET_CURRENT_USER } from './actionTypes';
+
+
+/**
+ * set current user action creator
+ * @export
+ * @param {object} user
+ * @returns {object} payload containing type and user object
+ */
+export function setCurrentUser(user) {
+  return {
+    type: SET_CURRENT_USER,
+    user
+  };
+}
+
+
+/**
+ * logout action creator
+ * @export
+ * @returns {function}
+ */
+export function logout() {
+  return (dispatch) => {
+    localStorage.removeItem('jwtToken');
+    localStorage.removeItem('user');
+    setAuthorizationToken(false);
+    dispatch(setCurrentUser({}));
```

Comment:
This is meant to un-set the user in state; A better name could be giving to this function.